src/dotnet/APIView/APIViewWeb/Repositories/CosmosSamplesRevisionsRepository.cs

```diff
@@ -25,6 +29,72 @@ public async Task<IEnumerable<SamplesRevisionModel>> GetSamplesRevisionsAsync(st
             return await GetSamplesRevisionsFromQueryAsync($"SELECT * FROM UsageSamples c WHERE c.ReviewId = '{reviewId}' AND c.IsDeleted = false");
         }
 
+        public async Task<PagedList<SamplesRevisionModel>> GetSamplesRevisionsAsync(ClaimsPrincipal user, PageParams pageParams, FilterAndSortParams filterAndSortParams)
+        {
+            var queryStringBuilder = new StringBuilder(@"SELECT * FROM Revisions c");
+            queryStringBuilder.Append($" WHERE c.IsDeleted = {filterAndSortParams.IsDeleted.ToString().ToLower()}");
```

Comment:
curious, why would someone want the samples for deleted revisions? should we instead just always say `WHERE c.IsDeleted = No` (ignore format)

Reply:
We might want to be able to restore deleted sample. No way to do this now in the UI though. But this enables it through the API.